vm/src/pyobject.rs

```diff
@@ -1407,6 +1407,7 @@ where
 
 pub trait PyClassDef {
     const NAME: &'static str;
+    const TP_NAME: &'static str;
```

Comment:
```suggestion
    const TP_NAME: &'static str = Self::NAME;
```

Reply:
I don't think this default value is good. `tp_name` in CPython includes module name.

Reply:
Alright, this looks fine to me then.